src/Atmos/Model/radiation.jl

```diff
@@ -54,12 +55,12 @@ function flux_radiation!(m::StevensRadiation, flux::Grad, state::Vars,
   z = aux.orientation.Φ/grav
   Δz_i = max(z - m.z_i, -zero(FT))
   # Constants
-  cloud_top_cooling = m.F_0 * exp(-aux.∫dnz.radiation.∂κLWP)
+  cloud_top_cooling  = m.F_0 * exp(-aux.∫dnz.radiation.∂κLWP)
   cloud_base_warming = m.F_1 * exp(-aux.∫dz.radiation.∂κLWP)
   free_troposphere_cooling = m.ρ_i * FT(cp_d) * m.D_subsidence * m.α_z * ((cbrt(Δz_i))^4 / 4 + m.z_i * cbrt(Δz_i))
-  F_rad = cloud_base_warming + cloud_base_warming + free_troposphere_cooling
-  flux.ρe += SVector(FT(0), 
-                     FT(0), 
+  F_rad = cloud_base_warming + cloud_top_cooling + free_troposphere_cooling
+  flux.ρe -= SVector(FT(0),
```

Comment:
why should this be negative?

Reply:
@smarras79 ?

Reply:
@simonbyrne @charleskawczynski it's because it is a simple radiative cooling model that takes heat away from the cloud top. It is not a full radiation scheme. 
@tapios please, confirm.

Reply:
These are longwave radiative energy fluxes, defined positive upward. So they should enter the energy flux as fluxes that are positive upward, hence with `+=`. Heating/cooling rates come from taking the convergence (minus vertical derivative). This gives cooling where fluxes increase with height (above the cloud).

Reply:
If this is the case, then is this a bug in the branch in PR #610? @smarras79